### ISSUE TITLE
finetune new data :Accuracy = 0

### ISSUE THREAD

**Issue by Bonsen** [tensorflow/models#1203]
I use my own data( 2 classes ) follow flowers format (download_and_convert_flowers.py),
and
I train inceptionv3 ,and get:  
eval/Accuracy[0.003235294]
eval/Recall@5[0.10088235]

if I should set class_num in train_image_classifier.py or download_and_convert_flowers.py??
I think it may be the class problem.
